### DISCUSSION TITLE
How can I resolve the warning that appears in the runlog

### DISCUSSION THREAD

**Discussion by caolz** [deepmodeling/deepmd-kit#430] [Q&A]
“WARNING: Radial neighbor list length of type WARNING: Radial neighbor list length of type WARNING: Radial neighbor list length of type 0 is not enough.” 

My structure has only one element.

Here are the settings：
            "type":             "se_a",
            "sel":              [20],
            "rcut_smth":        3.80,
            "rcut":             6.00,
            "neuron":           [25, 50, 100],

**Comment by njzjz** [comment]
see #108

**Reply by caolz** [reply]
Thank you very much.
